src/lib/containers/entity_finder/details/view/DetailsViewRow.tsx

```diff
@@ -110,87 +125,98 @@ export const DetailsViewRow: React.FunctionComponent<DetailsViewRowProps> = ({
       }}
     >
       {selectButtonType !== 'none' && (
-        <td className="IsSelectedColumn">
-          {!isDisabled && selectButtonType === 'checkbox' && (
-            <Checkbox
-              label=""
-              id=""
-              className="SRC-pointer-events-none"
-              checked={isSelected}
-              onChange={() => {}}
-            />
-          )}
-          {!isDisabled && selectButtonType === 'radio' && (
-            <RadioGroup
-              className="SRC-pointer-events-none"
-              options={[{ label: '', value: 'true' }]}
-              value={isSelected.toString()}
-              id=""
-              onChange={() => {}}
-            />
-          )}
+        <td className="IsSelectedColumn" aria-label="is-selected">
```

Comment:
Should we be moving the scss in this direction too (to use the aria attributes)?
```
[aria-label="is-selected"] {
  //styles
}
```